packaging/specifiers.py

```diff
@@ -453,12 +453,7 @@ def _compare_compatible(self, prospective, spec):
         # ignore post and dev releases and we want to treat the pre-release as
         # it's own separate segment.
```

Comment:
Would need to update this comment as well.

Reply:
Updated.